ttt/computer_player.py

```diff
@@ -0,0 +1,10 @@
+import random
+from ttt.player import Player
+
+
+class ComputerPlayer(Player):
+    def __init__(self, name, marker):
+        super().__init__(name, marker)
+
+    def get_move(self, spaces, console=None):
```

Comment:
Which of the SOLID principles does this method indicate you might have a violation of?